centos_ci.sh

```diff
@@ -304,12 +312,17 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags ~coolstore -format pretty"
+  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags proxy -format pretty"
```

Comment:
> This is just to test if this passes centos-ci, on merge it should be probably changed to MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags ~coolstore,~proxy -format pretty"

The problem is really the time it takes to run all this and the fact that we run it as part of pull request builds which should complete in a reasonable time. I guess _proxy_ in itself runs in a reasonable time. _proxy_ + _basic_ might become a border case. I guess we can see how long it takes. _coolstore_ is a no-go for now, since provisioning takes 30min+.

That said, why do we not add the nightly build at this stage. The default could then just run _basic_ and in the nightly we build all features we have.